app/assets/stylesheets/components/user_settings.scss

```diff
@@ -13,7 +13,6 @@
   .profile-pic {
     margin-right: 32px;
     img {
-      border-radius: 50%;
```

Comment:
@renziver why was this removed? its needed for the avatar on the settings page. Can we put it back in please?